src/utils/colors-utils.ts

```diff
@@ -3,7 +3,7 @@ import { COLOR_STYLES, contentColors } from "./colors-vars-map";
 const ColorUtils = {
   modes: COLOR_STYLES,
   contentColors,
-  getMondayColorAsStyle: (color, mode = COLOR_STYLES.REGULAR, withVar = true) => {
+  getMondayColorAsStyle: (color: string, mode = COLOR_STYLES.REGULAR, withVar = true) => {
```

Comment:
is any color acceptable here? 
if not, is there a quick way to type it? (should it be the property values of `contentColorsByName`?)